api/NetworkInformation.json

```diff
@@ -178,13 +178,13 @@
               "version_added": "38"
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "61"
```

Comment:
Do you have a source for this? Implementation bug or the like?

Reply:
I've just tested on my firefox android app. version 61.0.2

Reply:
Okay cool, that gives us the info that it's supported (`true`), but not that is was added in 61. Might have been added in an earlier version. So, either we're trying to find out that version or we mark this as `true`.